### PR TITLE
Create test for unicode symbols

### PR DESCRIPTION
**Pull request by HoffmannP**
This test is created b/c the same test with extensions would fail (see #317)

### CODE REVIEW

File: tests/unicode.php [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+# insert unicode via executeString
+$jscript->executeString("var execStr = {unicode: '" . $unicode . "'}");
+
+# return  to php
+$jscript->executeString("values = {}");
+$jscript->executeString("values['snapshot'] = snapshot.unicode");
+$jscript->executeString("values['php'] = php.unicode");
+$jscript->executeString("values['execStr'] = execStr.unicode");
+$values = $jscript->executeString("values", V8Js::FLAG_FORCE_ARRAY);
```
Comment (stesie) [inline tests/unicode.php]: The second argument to `executeString` is the script (file) name, not flags (which is the 3rd arg)

File: tests/unicode.php [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+--SKIPIF--
+<?php require_once(dirname(__FILE__) . '/skipif.inc'); ?>
+--FILE--
+<?php
+
+# maybe more characters (e.g. from http://www.ltg.ed.ac.uk/~richard/unicode-sample.html?)
+$unicode = 'äöüßÜÄÖÜß€áàâÁÀÂµ²³▁▂▃▄▅▆▇█    ㌀ ㌁ ㌂ ㌃';
+
+# insert unicode via snapshot
+$snapshot = V8Js::createSnapshot("var snapshot = {unicode: '" . $unicode . "'}");
```
Comment (stesie) [inline tests/unicode.php]: Keep in mind that V8Js can be compiled with snapshot support turned off.

You might want to create a second test that focuses on testing unicode support in combination with snapshots (and just skip it if V8 doesn't have support for snapshots)

**Comment by stesie** [comment]
Thanks for your pull request, I like the idea to just test that.

However this way we would force users of V8Js to install V8 with snapshot-support turned on (which generally is a good idea, but so far wasn't enforced), so I ask you to test this more specifically.

If you need help regarding this, feel free to ask :-)

**Comment by HoffmannP** [comment]
So I wrote a check if snapshots are supported but don't know if it works, could you confirm?

    
    define('V8_WITH_SNAPSHOT', method_exists("V8Js", "createSnapshot"));
    

btw: would you recommend adding a (failing) unicode test for presumably outdated extensions based on the same bug or is that considered a "no fix"?

**Pull request merged**